nfs/defaults.yaml

```diff
@@ -0,0 +1,13 @@
+nfs:
+  mount_opts: "vers=3"
```

Comment:
This could be blank, so linux distribution default is used. RHEL default is NFSv4